pkgs/tools/networking/burpsuite/community.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchurl, openjdk, runtimeShell, unzip, chromium }:
```

Comment:
Need to follow commit format and rename the PR to something like `burpsuite-pro: init at 2022.3.4 `, read [`CONTRIBUTING`](https://github.com/NixOS/nixpkgs/blob/866a9a0c5a4371d69c160e8cecbaeb0b44b0e645/CONTRIBUTING.md).